examples/src/examples/graphics/render-cubemap.tsx

```diff
@@ -168,35 +137,36 @@
 
         // use cubemap which is generated by cubemapRenderer instead of global skybox cubemap
         shinyMat.useSkybox = false;
+        // @ts-ignore engine-tsd
         shinyMat.cubeMap = shinyBall.script.cubemapRenderer.cubeMap;
 
-        // make it shiny without diffuse component        
+        // make it shiny without diffuse component
         shinyMat.metalness = 1;
         shinyMat.useMetalness = true;
         shinyMat.update();
 
         // create few random primitives in the world layer
-        var entities = [];
-        var shapes = ["box", "cone", "cylinder", "sphere", "capsule"];
-        for (var i = 0; i < 6; i++) {
-            var shapeName = shapes[Math.floor(Math.random() * shapes.length)];
-            var color = new pc.Color(Math.random(), Math.random(), Math.random());
+        const entities: { rotate: (arg0: number, arg1: number, arg2: number) => void; }[] | pc.Entity[] = [];
```

Comment:
just curious about why this line got so complicated?

Reply:
Oh this was created by VSCode's autotyping for TypeScript. I guess it inferred that entities each contain a rotate method (used on line 215). I'll remove this and just keep the pc.Entity[] type.